Mesh_3/include/CGAL/Labeled_mesh_domain_3.h

```diff
@@ -565,6 +653,30 @@ class Labeled_mesh_domain_3
    *                            bound, relative to the diameter of the box of the image.}
    *     \cgalParamDefault{FT(1e-3)}
    *   \cgalParamNEnd
+   *
+   *   \cgalParamNBegin{detect_features}
+   *    \cgalParamDescription{ a functor that can be passed to
+   *      `std::function<
+   *        std::vector<std::vector<Point_3>>
+   *        (const CGAL::Image_3&, Mesh_domain_with_polyline_features_3<Labeled_mesh_domain_3>&)>`
+   *      and that returns a range of detected polyline features for feature protection.
+   *      Polyline features are added to the domain for further feature protection.
+   *      See \ref PkgMesh3FeatureDetection for available functors.}
+   *    \cgalParamDefault{CGAL::Null_functor()}
+   *    \cgalParamExtra{The return type of the function depends on whether this parameter
+                        or `input_features` are provided or not.}
+   *    \cgalParamExtra{If `weights` is provided, this parameter is ignored}
+   *   \cgalParamNEnd
+   *
+   *   \cgalParamNBegin{input_features}
+   *    \cgalParamDescription{ a `Range` of polyline features, represented as `Range`s of `Point_3`.
+   *         Polyline features are added to the domain for further feature protection.}
+   *    \cgalParamDefault{`std::vector<std::vector<Point_3>>()`}
+   *    \cgalParamExtra{The return type of the function depends on whether this parameter
+                        or `input_features` are provided or not.}
+   *    \cgalParamExtra{If `weights` is provided, this parameter is ignored}
```

Comment:
Because we only have experimental code that is able to deal with features and weights, and for now we are not sure that it is reliable. Hopefully coming soon :-)